File: app-src/src/bin/client_gen/csharp/Debug.cs
using System.IO;

namespace AiCup2019
{
    public class Debug
    {
        private BinaryWriter writer;
        public Debug(BinaryWriter writer)
        {
            this.writer = writer;
        }
        public void Draw(Model.CustomData customData)
        {
            new Model.PlayerMessageGame.CustomDataMessage(customData).WriteTo(writer);
            writer.Flush();
        }
    }
}

File: app-src/src/bin/client_gen/go/debug.go
package main

import . "aicup2019/model"
import "bufio"

type Debug struct {
	Writer *bufio.Writer
}

func (debug Debug) Draw(data CustomData) {
	PlayerMessageGameCustomDataMessage {
		Data: data,
	}.Write(debug.Writer)
	err := debug.Writer.Flush()
	if err != nil {
		panic(err)
	}
}